applications/fluid_structure_interaction_precice/perpendicular_flap/fluid-exadg/input.json

```diff
@@ -0,0 +1,23 @@
+{
+    "General": {
+        "Precision": "double",
+        "Dim": "2",
+        "IsTest": "false"
+    },
+    "SpatialResolution": {
+        "DegreeFluid": "2",
+        "DegreeStructure": "3",
+        "RefineFluid": "1",
+        "RefineStructure": "3"
+    },
+    "preciceConfiguration": {
+        "preciceConfigFile": "../precice-config.xml",
+        "Physics": "Fluid",
+        "ParticipantName": "Fluid",
+        "ReadMeshName": "Fluid-Mesh-read",
+        "WriteMeshName": "Fluid-Mesh-write",
+        "WriteDataSpecification": "values_on_quads",
+        "ReadDataName": "Displacement",
```

Comment:
Is the velocity vector also exchanged as you mention in the description of this PR? Or, put differently, why isn't there a parameter "ReadDataName": "Velocity"?